pina/geometry/simplex.py

```diff
@@ -0,0 +1,218 @@
+import torch
+
+from pina.geometry import CartesianDomain
+from .location import Location
+from pina import LabelTensor
+from ..utils import check_consistency
+
+
+class SimplexDomain(Location):
+    """PINA implementation of a Simplex."""
+
+    def __init__(self, simplex_dict, labels, sample_surface=False):
+        """
+        :param simplex_dict: A dictionary with dict-key a string representing
+            the input variables for the pinn, and dict-value a list
```

Comment:
input variables for the problem

Reply:
Changed the documentation to reflect this and that the labels are necessary for determining order of coordinates in point